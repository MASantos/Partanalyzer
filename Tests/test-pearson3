Bapx1_2343.1 Alx3_3418.2 0.472933
Alx3_3418.2 Alx4_1744.1 0.904824
Alx3_3418.2 Arx_1738.2 0.927179

