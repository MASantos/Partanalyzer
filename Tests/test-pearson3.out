Alx3_3418.2	Alx4_1744.1	0.904824
